concepticondata/conceptlists/Zalizniak-2020-2591.tsv

```diff
@@ -0,0 +1,2592 @@
+Zalizniak-2020-2591-827	827	"secret; covert"	SECRET	1121	827:—:1 827:—:1	shift2752 shift2790	2	2
+Zalizniak-2020-2591-828	828	scorpion	SCORPION	1538	828:→:2 828:—:1	shift1587 shift4284	2	3
+Zalizniak-2020-2591-829	829	sand	SAND	671	829:→:1 829:→:1	shift2113 shift2114	2	2
+Zalizniak-2020-2591-830	830	sack, bag			830:→:10 830:→:2	shift986 shift1754	2	12
+Zalizniak-2020-2591-831	831	rye (Secale)	RYE	23	831:—:3 831:→:2	shift3430 shift3198	2	5
+Zalizniak-2020-2591-832	832	rust (n.)	RUST	3064	832:→:1 832:—:0	shift1535 shift1534	2	1
+Zalizniak-2020-2591-833	833	rural			833:→:6 833:→:0	shift3125 shift3477	2	6
+Zalizniak-2020-2591-834	834	runner			834:—:4 834:→:2	shift3815 shift3230	2	6
+Zalizniak-2020-2591-835	835	rumour	RUMOUR	63	835:—:13 835:—:2	shift625 shift1094	2	15
+Zalizniak-2020-2591-836	836	rude, impolite	RUDE	1412	836:—:5 836:—:1	shift1435 shift1597	2	6
+Zalizniak-2020-2591-837	837	rowan			837:—:1 837:—:1	shift2461 shift2444	2	2
+Zalizniak-2020-2591-838	838	round (adj.)	ROUND	1395	838:→:7 838:→:1	shift1143 shift3269	2	8
+Zalizniak-2020-2591-839	839	rotten	ROTTEN	1728	839:→:1 839:→:1	shift3049 shift3048	2	2
+Zalizniak-2020-2591-840	840	river	RIVER	666	840:—:3 840:—:2	shift1451 shift3700	2	5
+Zalizniak-2020-2591-841	841	rich	RICH	712	841:→:3 841:—:1	shift1233 shift4113	2	4
+Zalizniak-2020-2591-842	842	rhinoceros	RHINOCEROS	2591	842:↔:3 842:→:1	shift1447 shift2146	2	4
+Zalizniak-2020-2591-843	843	respect (n.)	RESPECT (NOUN)	1912	843:—:2 843:—:1	shift51 shift205	2	3
+Zalizniak-2020-2591-844	844	relative (n.)	RELATIVES	1644	844:—:1 844:—:1	shift3779 shift3873	2	2
+Zalizniak-2020-2591-845	845	ray			845:—:1 845:—:1	shift3959 shift607	2	2
+Zalizniak-2020-2591-846	846	raven	RAVEN	2543	846:—:1 846:—:1	shift1484 shift2934	2	2
+Zalizniak-2020-2591-847	847	rare	RARE	209	847:↔:3 847:—:2	shift2567 shift1520	2	5
+Zalizniak-2020-2591-848	848	rag, duster (?)	RAG	332	848:→:3 848:→:1	shift1537 shift4199	2	4
+Zalizniak-2020-2591-849	849	pus	PUS	1808	849:—:1 849:—:1	shift1296 shift1298	2	2
+Zalizniak-2020-2591-850	850	pupil (of an eye)	PUPIL	1658	850:—:1 850:—:1	shift3906 shift670	2	2
+Zalizniak-2020-2591-851	851	profligate, womanizer	PROFLIGATE	115	851:—:3 851:—:1	shift119 shift261	2	4
+Zalizniak-2020-2591-852	852	priest (Christianity)	PRIEST	852	852:→:3 852:→:1	shift3203 shift3797	2	4
+Zalizniak-2020-2591-853	853	price	PRICE	657	853:→:3 853:—:2	shift1022 shift675	2	5
+Zalizniak-2020-2591-854	854	present, gift	GIFT	2982	854:→:2 854:→:1	shift3588 shift3589	2	3
+Zalizniak-2020-2591-855	855	precipice, abyss	PRECIPICE	618	855:↔:4 855:→:3	shift3392 shift3393	2	7
+Zalizniak-2020-2591-856	856	potato	POTATO	593	856:—:4 856:→:1	shift4260 shift149	2	5
+Zalizniak-2020-2591-857	857	pot	POT	1462	857:—:2 857:—:1	shift257 shift2168	2	3
+Zalizniak-2020-2591-858	858	poppy (Papaver somniferum)			858:→:1 858:—:1	shift2977 shift2978	2	2
+Zalizniak-2020-2591-859	859	people, nation	PEOPLE	789	859:—:6 859:—:2	shift1156 shift263	2	8
```

Comment:
I mapped it to PEOPLE because the description includes the "nation" part:
A group of persons forming or belonging to a particular nation, class, ethnic group, country, family, etc.